ui/src/main/webapp/src/app/misc/about.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, OnInit, AfterViewInit} from "@angular/core";
+
+@Component({
+    templateUrl: './about.component.html',
+    styles: [`
+        /deep/ img.windupContributor {
+            width: 35px;
+            margin: 4px;
+        }
+    `]
+})
+export class AboutPageComponent implements OnInit, AfterViewInit {
+
+    ngOnInit():any {
+    }
+
+    ngAfterViewInit(): void {
+        var divTarget = $("#windup-contributors");
```

Comment:
There is the same code duplicated in the template. Why? Why do you use jQuery API here, I think it would be better to stick to Angular Http.
Also DOM manipulation outside Angular (using jQuery) might cause some issues. This component doesn't really have any binding, but be aware of that.